### ISSUE TITLE
filter "cancelled invoice" 

### ISSUE THREAD

**Issue by BeckeBauer** [kevinpapst/kimai2#3411]
Kimai allows to filter invoices as regards the statuses "new", "pending" and "paid". It would be great to to filter "cancelled" invoices as well.

**Comment by kevinpapst** [comment]
Good point 👍  I think "canceled" was added later and I forgot to add it to the search filter.

**Comment by BeckeBauer** [comment]
Great, thank you! 👌
